hail/src/main/scala/is/hail/expr/ir/EmitStream.scala

```diff
@@ -820,22 +829,55 @@ object EmitStream {
             }
           }
 
+        case x@StreamGrouped(a, size) =>
+          val innerType = coerce[PCanonicalStream](coerce[PStream](x.pType).elementType)
+          val optStream = emitStream(a, env)
+          val optSize = COption.fromEmitCode(emitIR(size))
+          val xS = mb.newLocal[Int]("st_n")
```

Comment:
Is it safe to hard code the name of locals? Don't you need to generate an identifier in case this one conflicts with a name that's in use? I'm not sure, just asking.

Reply:
Yeah, these names are only for debugging. The JVM doesn't have names for locals (it just indexes into the array of locals), and we only refer to the local using the scala variable (`xS`), not the name.

Reply:
Actually, lir locals may become either JVM locals or class fields. In the latter case, I assume this name will be used as a base for the field name. But regardless, all uses of `newLocal` have hardcoded names like this.